app/games/simulation-engine.ts

```
@@ -0,0 +1,516 @@
+      randomValue -= eventType.probability;
+      if (randomValue <= 0) {
+        selectedEventType = eventType.type;
+        break;
+      }
+    }
+    
+    const isOurEvent = Math.random() < ourAdvantage;
+    
+    switch (selectedEventType) {
+      case 'goal':
+        const scorer = this.selectRandomPlayer('attacker');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)